### PR TITLE
Steering group minutes, 2024-09-23

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2024/2024-09-24-steering.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: "HSF Steering Group Meeting, 24 September 2024"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Eduardo Rodrigues, Benedikt Hegner, Pere Mato, Stefan Roiser
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply organization/_posts/2024/2024-09-24-steering.md]: Was in the very next commit 😉 !

File: organization/_posts/2024/2024-09-24-steering.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+- [ ] Michel, Graeme to reach out to activity conveners
+
+## HSF and SFT reporting to WLCG/LHCC
+
+- Eduardo presented to the LHCC on Sep. 10th the "HSF/community Software Report".
+    - Very well received.
+    - The LHCC commends the HSF for its evolution and ongoing reorganisation efforts. It also congratulates us on our software stewardship role within the community
+(e.g., successful PyHEP initiatives, efforts to explore the role of Julia in HEP, the highly appreciated training programme).
+- Eduardo submitted the part "HSF and Community Software Activities" for the October RRB report. (Thank you as ever for the comments/inputs.)
+    - This incorporated paragraphs not just on the HSF but also from community projects such as IRIS-HEP and SWIFT-HEP.
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply organization/_posts/2024/2024-09-24-steering.md]: Rephrased

File: organization/_posts/2024/2024-09-24-steering.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
+---
+title: "HSF Steering Group Meeting, 24 September 2024"
```
Comment (eduardo-rodrigues) [inline organization/_posts/2024/2024-09-24-steering.md:2]:
```suggestion
title: "HSF Steering Group Meeting, 23 September 2024"
```

**Review by eduardo-rodrigues — APPROVED** [review]
Thank you.

**Pull request merged**